main.go

```diff
@@ -37,6 +37,7 @@ var (
 	timeGranularity = flag.String("granularity", "day", "how often we expect to append new data")
 	streamStart     = flag.String("streamStart", "", "The start of the streamed data. Only used if granularity='stream'. Used to ensure idempotency")
 	streamEnd       = flag.String("streamEnd", "", "The end of the streamed data. Only used if granularity='stream'. Used to ensure idempotency")
+	isTargetPT      = flag.Bool("targetPT", false, "Whether or not the target data time column is in PT.")
```

Comment:
this also would give us the flexibility of ending on `America/New_York` in the future if we need to / others who might want to use this code